wlauto/result_processors/cpustate.py

```diff
@@ -102,6 +102,11 @@ class CpuStatesProcessor(ResultProcessor):
                   Create a CSV with the timeline of core power states over the course of the run
                   as well as the usual stats reports.
                   """),
+        Parameter('create_utilization_timeline', kind=bool, default=False,
+                  description="""
+                  Create a CSV with the timeline of cpu(s) utilisation over the course of the run
+                  as well as the usual stats reports.
+                  """),
 
```

Comment:
Can you expand this description to explain that it is an normalised average of frequencies. When you look at the raw CSV you only see a few values and this can be confusing until a sliding window average is performed